### ISSUE TITLE
Flatpak version of RttR doesn't see S2GE files

### ISSUE THREAD

**Issue by Piotr-Wieczorek** [Return-To-The-Roots/s25client#1685]
![Zrzut ekranu z 2024-07-24 16-18-13](https://github.com/user-attachments/assets/776103c7-870d-4788-a873-6e5c29dc60a0)

**Comment by Flow86** [comment]
Hi, afaik flatpak is not officially supported/maintained by us. can you give us some more insights? You can have a look into the logfile the game creates, maybe this gives more hints whats missing.

**Comment by Athemis** [comment]
Flatpak does prevent direct access to your home directory and instead uses a special directory which is visible to the application. For the RttR Flatpak this is located here:

`/home/<your_username>/.var/app/info.rttr.Return-To-The-Roots/.s25rttr/`

This is where the "S2" folder resides in which you have to put your game data (see screenshot below):

![Screenshot_20240726_121938](https://github.com/user-attachments/assets/40d248ef-b9e8-401f-97b0-3b7207749284)

To summarize: If using the Flatpak, don't put your game data into `/home/<user>/.s25rttr` but into the Flatpak-specific folder.

**Comment by Flamefire** [comment]
Looks like this has been fixed in the Flatpak package.

There is a comment
> This way we have to patch the AppStream metadata in a way that we cannot upstream. 

So I conclude that there isn't anything that we can do here as this is an issue with Flatpak.

Great to see this used and included broadly :)
